Makefile

```
@@ -5,16 +5,48 @@ include $(shell test -f .tardigrade-ci || curl -sSL -o .tardigrade-ci "https://r
 
 pytest/install:
 	@ $(MAKE) install/pip/$(@D) PYPI_PKG_NAME=$(@D)
-	@ python -m pip install -r lambda/tests/requirements_dev.txt
+	@ python -m pip install \
+		-r lambda/tests/requirements_dev.txt \
+		-r tests/requirements_test.txt
 
+.PHONY: python/deps
 python/deps:
 	@ echo "[$@] Installing package dependencies"
 	@ python -m pip install -r lambda/src/requirements.txt
 
+.PHONY: python/test
 python/test: | guard/program/pytest
 python/test:
 	@ echo "[$@] Starting Python tests"
-	pytest
+	pytest lambda/tests
 	@ echo "[$@]: Tests executed!"
 
-.PHONY: python/deps python/test
+.PHONY: terraform/pytest
+terraform/pytest: | guard/program/terraform guard/program/pytest
+	@ echo "[$@] Starting test of Terraform lambda installation"
+	@ echo "[$@] Terraform 'apply' command is slow ... be patient !!!"
+	pytest tests
+	@ echo "[$@]: Completed successfully!"
+
+.PHONY: localstack/pytest localstack/up localstack/down localstack/clean
+localstack/pytest: | guard/program/terraform guard/program/pytest
+	@ echo "[$@] Running Terraform tests against LocalStack"
+	DOCKER_RUN_FLAGS="--network host --rm" \
+		$(MAKE) docker/run target=terraform/pytest
+	@ echo "[$@]: Completed successfully!"
+
+localstack/up: | guard/program/terraform guard/program/pytest
+	@ echo "[$@] Starting LocalStack"
+	docker-compose -f tests/docker-compose-localstack.yml up --detach
+
+localstack/down: | guard/program/terraform guard/program/pytest
+	@ echo "[$@] Stopping and removing LocalStack container"
+	docker-compose -f tests/docker-compose-localstack.yml down --rm all
```

Ok, one more thing. It's a bit annoying for localstack/down to _always_ remove the localstack image. Suggest maybe move the removal of the image to localstack/clean?

```suggestion
	docker-compose -f tests/docker-compose-localstack.yml down
```

---

Agreed.  Done.